### PR TITLE
Harolddavis/ch117631/cell ui should adapt the beta branch colors

### PR DESCRIPTION
**Pull request by HaroldFS**
This PR addresses the following:
* Cell UI should adapt the beta branch colors
* Text labels should be white
* Background color should be navy

### CODE REVIEW

File: swift/ios-shoppe-demo/Views/StoreViewController.swift [hunk 1]
```diff
@@ -50,7 +50,7 @@ class StoreViewController: UICollectionViewController, UICollectionViewDelegateF
         navigationItem.title = "iOS Shoppe"
         navigationItem.rightBarButtonItem = barCartButton
 
-        collectionView.backgroundColor = .white
+        collectionView.backgroundColor = UIColor().fsBackground()
```
Comment (van-fs) [inline swift/ios-shoppe-demo/Views/StoreViewController.swift:53]: Just curious, which color is this?  Is it one from the swatch?

**Review by van-fs — APPROVED** [review]
LGTM

File: swift/ios-shoppe-demo/Views/StoreViewController.swift (hunk 1 above)
Comment (HaroldFS) [reply swift/ios-shoppe-demo/Views/StoreViewController.swift:53]: I'm not sure what the swatch is but its a dark navy color I got from our power point intro slides.

Comment (van-fs) [reply swift/ios-shoppe-demo/Views/StoreViewController.swift:53]: Probably the same, but you might check https://fullstory.slack.com/files/UNTS3T26Q/F011MCK62F9/image.png and then convert the hex to RGB.  It's probably `.enterprise` but naming it in a code comment would help identify which color the RGB value applies to.

Comment (HaroldFS) [reply swift/ios-shoppe-demo/Views/StoreViewController.swift:53]: I will post a separate pr for this :)

**Pull request merged**